pypeit/core/coadd2d.py

```diff
@@ -788,19 +792,28 @@ def reduce(self, psuedo_dict, show=None, show_peaks=None):
 
         # Make changes to parset specific to 2d coadds
         parcopy = copy.deepcopy(self.par)
-        parcopy['scienceimage']['trace_npoly'] = 3        # Low order traces since we are rectified
+        parcopy['scienceimage']['findobj']['trace_npoly'] = 3        # Low order traces since we are rectified
         #parcopy['scienceimage']['find_extrap_npoly'] = 1  # Use low order for trace extrapolation
-        redux = reduce.instantiate_me(sciImage, self.spectrograph, psuedo_dict['tslits_dict'], parcopy, psuedo_dict['tilts'],
-                                      ir_redux=self.ir_redux, objtype = 'science_coadd2d', det=self.det, binning=self.binning)
+        # Instantiate Calibrations class
+        caliBrate = calibrations.MultiSlitCalibrations(None, parcopy['calibrations'], self.spectrograph)
```

Comment:
Remove the dependency on calibrations here. There is simply no need to create a calibrations object and then pass that into reduce, when reduce only needs two things, tslits_dict, and tilts. You just added four unnecessary lines of code and added an unnecessary dependency.